pkg/cri/sbserver/image_pull.go

```diff
@@ -92,7 +93,20 @@ import (
 // contents are missing but snapshots are ready, is the image still "READY"?
 
 // PullImage pulls an image with authentication config.
-func (c *criService) PullImage(ctx context.Context, r *runtime.PullImageRequest) (*runtime.PullImageResponse, error) {
+func (c *criService) PullImage(ctx context.Context, r *runtime.PullImageRequest) (_ *runtime.PullImageResponse, err error) {
+	var domain string
```

Comment:
Let's remove this variable and the parsing logic now that we're no longer using it.

Reply:
Updated.